src/EnergyPlus/NodeInputManager.cc

```diff
@@ -109,8 +110,8 @@ namespace NodeInputManager {
 		NumCheckNodes = 0;
 		MaxCheckNodes = 0;
 		NodeVarsSetup = false;
-		NodeWetBulbRepReq.deallocate();
```

Comment:
@EnergyArchmage Since you've been working in CalcModeNodeInfo recently, please review this deletion.  Does it look OK?